bootstrap/sql/migrations/native/1.4.0/mysql/postDataMigrationSQLScript.sql

```diff
@@ -362,3 +362,23 @@ set json = JSON_INSERT(
 )
 WHERE name = 'tableRowInsertedCountToBeBetween';
 -- End of Test Definition Parameter Definition Validation Migration
+
```

Comment:
you can just do delete from openmetadata_settings where configType = 'emailConfiguration'; and let the server start SettingCache.java let this build no (less chance to make mistakes)

Reply:
we will need the old config values that are stored in customLogoConfiguration